tests/integration/features/integration/dashboard.feature

```diff
@@ -0,0 +1,35 @@
+Feature: integration/dashboard
+  Background:
+    Given user "participant1" exists
+    Given user "participant2" exists
+
+  Scenario: User gets the available dashboard widgets
+    When user "participant1" sees the following entry when loading the list of dashboard widgets (v1)
+      | id     | title         | icon_class          | icon_url         | widget_url                                   | item_icons_round | order | buttons |
+      | spreed | Talk mentions | dashboard-talk-icon | img/app-dark.svg | http://localhost:8080/index.php/apps/spreed/ | true             | 10    | [{"type":"more","text":"More unread mentions","link":"http://localhost:8080/index.php/apps/spreed/"}] |
```

Comment:
I think that is good to replace the URL by a placeholder representing the place of the base URL. Today the URL is fixed, but this can change in other environments.
For example, using `<baseUrl>` instead of `http://localhost:8080` and  in the method of this sentence do the magic handling the key "link".